include/crm/common/output_internal.h

```diff
@@ -161,6 +161,17 @@ pcmk__output_t *pcmk__mk_xml_output(char **argv);
 #define PCMK__SUPPORTED_FORMAT_TEXT { "text", pcmk__mk_text_output, pcmk__text_output_entries }
 #define PCMK__SUPPORTED_FORMAT_XML  { "xml", pcmk__mk_xml_output, pcmk__xml_output_entries }
 
+/*!
+ * \brief Enable various tweaks to the XML formatted output
```

Comment:
```
/*!
 * \internal
 * \brief ...
```

Reply:
Fixed, also changed this comment to make it a little clearer that you only enable a single thing at a time.